src/native/fs/fs_napi.cpp

```diff
@@ -35,7 +35,10 @@ namespace noobaa
 
 DBG_INIT(0);
 
-static int (*dlsym_gpfs_fcntl)(gpfs_file_t file, void* arg) = 0;
```

Comment:
undelete dlsym_gpfs_fcntl

Reply:
I had this line in but testing failed because of unused define.

Reply:
done

Reply:
I assume it's not complaining on unused line now, right? resolving.